src/index.html

```diff
@@ -24,7 +24,7 @@ <h2>Add your score</h2>
             <div class="right-inputs">
                 <input type="text" name="name" id="yourName" placeholder="Your name">
                 <input type="text" name="score" id="yourScore" placeholder="Your score">
```

Comment:
**Changes Required**
- [ ] This should be input type `number`. You don't want users to enter any characters as scores. Let us refactor it to only accept numbers.

Reply:
Well solved